core/domain/learner_progress_services.py

```diff
@@ -311,6 +311,50 @@ def mark_collection_as_incomplete(user_id, collection_id):
         _save_incomplete_activities(incomplete_activities)
 
 
+def add_collection_to_learner_playlist(
+        user_id, collection_id, position_to_be_inserted=None):
+    """This function checks if the collection exists in the completed list or
+    the incomplete list. If it does not exist we call the function in learner
+    playlist services to add the collection to the play later list.
+
+    Args:
+        user_id: str. The id of the user.
+        collection_id: str. The id of the collection to be added to the
+            learner playlist.
+        position_to_be_inserted: int|None. If this is specified the collection
+            gets inserted at the given position. Otherwise it gets added at the
+            end.
+
+    Returns:
```

Comment:
Just use one return value. Please check the rest of the PR.

Reply:
I have changed this and checked the PR. I found one other occasion where I had to change.